metricbeat/helper/prometheus/metric.go

```diff
@@ -212,3 +242,25 @@ func (m *infoMetric) GetValue(metric *dto.Metric) interface{} {
 func (m *infoMetric) GetField() string {
 	return ""
 }
+
+type opFilter struct {
+	labels map[string]string
+}
+
+func (o opFilter) Process(field string, value interface{}, labels common.MapStr) (string, interface{}, common.MapStr) {
```

Comment:
I wonder why `Hound` is not complaining here? Asking because this could use a comment.